packages/components/src/components/post-menu/post-menu.tsx

```diff
@@ -198,6 +198,8 @@ export class PostMenu {
         .filter(el => el.getAttribute('role') === 'menuitem')
         // For each menu item, get any focusable children (e.g., buttons, links)
         .flatMap(el => Array.from(getFocusableChildren(el)))
+        // Filter elements whose parent is hidden
+        .filter(el => window.getComputedStyle(el.parentElement).display !== 'none')
```

Comment:
It would be better to filter these elements within the `getFocusableChildren` helper as they are not actually focusable.